### DISCUSSION TITLE
layout view matrix

### DISCUSSION THREAD

**Discussion by hvelasquez78** [Budibase/budibase#10040] [Help]
I have a small app to record the tasks of the technical staff.

in a budibase database i have a table with the required data fields.

In a screen called record, insert a table block and a slide panel for the creation, modification or deletion of records.

I want to visualize in another screen called queries, the activities in the form of a matrix, but I cannot accommodate them in a uniform way......any idea or advice
![image2](https://user-images.githubusercontent.com/44243895/225435241-14b98dee-15c0-43c0-9906-cdf109b3b6a8.png)
![image1](https://user-images.githubusercontent.com/44243895/225435244-f1ce0bd8-991e-4ff1-8530-c776e67e0ece.png)

**Comment by melohagan** [comment]
Hey @hvelasquez78 

Have you tried any CSS to the FECHA container? You could try giving it a **min-width** attribute.
